cf-rd/src/main/java/org/eclipse/californium/tools/resources/KeyValuePair.java

```diff
@@ -67,7 +67,7 @@ public String getValue() {
 		return value==null ? "" : value;
 	}
 	
-	public int getIntValue() {
+	public int getIntValue() throws NumberFormatException {
```

Comment:
In "the most" other cases we use the approach to mention `RuntimeException` rather in the javadoc then to throws them explicit.

Reply:
Okay, I'll drop this commit.